__tests__/modules/student/commands/find-enrollment.command.mock.ts

```diff
@@ -0,0 +1,9 @@
+export const prismaServiceMock = {
+  student: {
+    findFirst: jest.fn(),
+  },
+};
+
+export const findEnrollmentFindFirstExpectedCall = (enrollment: string) => ({
+  where: { enrollment: enrollment },
+});
```

Comment:
Como os parâmetros dessa chamada são simples, nem precisa dessa função. Vamo criar ela só quando a chamada esperada for complexa ou tiver muita coisa que não muda